src/concepts/terminology.md

```diff
@@ -1,12 +1,236 @@
+    - [Client ID](#client-id)
+    - [Data Analyst](#data-analyst)
+    - [Data Practioner](#data-practioner)
+    - [Data Scientist](#data-scientist)
+    - [dataset](#dataset)
+        - [derived dataset](#derived-dataset)
+    - [DAU](#dau)
+    - [derived dataset](#derived-dataset)
+    - [Fenix](#fenix)
+    - [FxA](#fxa)
+    - [Glean](#glean)
+    - [GCP](#gcp)
+    - [GeoIP](#geoip)
+    - [ingestion](#ingestion)
+    - [Intensity](#intensity)
+    - [metric](#metric)
+    - [MAU](#mau)
+    - [Notebook Interface](#notebook-interface)
+    - [ping](#ping)
+    - [ping table](#ping-table)
+    - [pipeline](#pipeline)
+    - [probe](#probe)
+    - [profile](#profile)
+    - [query](#query)
+    - [retention](#retention)
+    - [Retention](#retention)
+        - [-Week New Profile Retention](#-week-new-profile-retention)
+        - [-Week Retention](#-week-retention)
+    - [schema](#schema)
+    - [session](#session)
+    - [structured ingestion](#structured-ingestion)
+    - [scheduled query](#scheduled-query)
+    - [sql.telemetry.mozilla.org](#sqltelemetrymozillaorg)
+    - [STMO](#stmo)
+    - [Telemetry](#telemetry)
+        - [WAU](#wau)
+
+<!-- /TOC -->
+
+## AET
+
+Account Ecosystem Telemetry; see the [PRD](https://docs.google.com/document/d/1yRLiD8JuaZIIaKhs6DhXEa7aH8jwOau5yW0kHaldFQU/edit#)
+
+## Analyst
+
+A data analyst is someone who looks at Mozilla data, identifies trends and other qualitative measurements in them, and creates charts and dashboards.
+
+## Amplitude
+
+A third-party product used by several teams within Mozilla for analysis of user events.
+
+## BigQuery
+
+[BigQuery](https://cloud.google.com/bigquery) is Google Cloud's managed data warehouse. Most of the data described on this site is stored and queried using BigQuery. See the [Accessing and working with BigQuery](https://docs.telemetry.mozilla.org/cookbooks/bigquery.html0) article for more details.
+
+## Client ID
+
+The id property of the Firefox browser returns the universally unique identifier of the Client object.
+
+## Data Analyst
+
+See [Analyst](#Analyst)
+
+
+## Data Practioner
+
+See [Analyst](#Analyst)
+
+## Data Scientist
+
+Data scientists at Mozilla are a team with a broad array of technical backgrounds and a core set of common professional skills: 1) applying statistical methods to noisy data to answer questions about what, how, or why something is happening; 2) transform unstructured data into usable metrics and models; and 3) augmenting strategic product and decision-making with empirical evidence created and curated by the team.
+
+## dataset
+
+A set of data, which includes ping data, derived datasets, etc.; sometimes it is used synonymously with “table”; sometimes it is used technically to refer to a [BigQuery](https://cloud.google.com/bigquery/docs/datasets-intro) dataset, which represents a container for one or more tables.
+
+### derived dataset
+
+A processed dataset, such as `main_summary` or the `clients_daily` dataset
+
+
```

Comment:
Seems to be a less refined version of the definition below, guessing this got included by mistake.

```suggestion
```